src/maxminddb.c

```diff
@@ -318,7 +322,7 @@ LOCAL int map_file(MMDB_s *const mmdb)
     if (INVALID_HANDLE_VALUE != fd) {
         CloseHandle(fd);
     }
-    if (INVALID_HANDLE_VALUE != mmh) {
+    if (INVALID_HANDLE_VALUE != mmh && NULL != mmh) {
```

Comment:
Didn't you already check whether `mmh` is `NULL` on line #284? I don't think that could change in between there and here.